metrotranscom/doorway
# fix: translation fixes

## Issue Overview

This PR addresses issues https://github.com/metrotranscom/doorway/issues/412 and https://github.com/metrotranscom/doorway/issues/411


## Description
The spreadsheet: https://docs.google.com/spreadsheets/d/1M1Jzp9JIAi6h4gIg7KgCRvc8knJ8DJpIc1ii7et2crA/edit#gid=0 has a grip of translations/copy updates to be made. This goes through and makes those updates throughout

## How Can This Be Tested/Reviewed?
It will require a bit of bouncing around but the spreadsheet should point you to where the copy update was made and that copy update should be made

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
